static/app/components/idBadge/baseBadge.tsx

```diff
@@ -43,7 +44,12 @@ export const BaseBadge = memo(
     const wrapperGap: ValidSize = avatarSize <= 14 ? 0.5 : avatarSize <= 20 ? 0.75 : 1;
 
     return (
-      <Wrapper className={className} gap={wrapperGap}>
+      <Wrapper
+        className={className}
+        css={css`
+          --wrapper-gap: ${space(wrapperGap)};
+        `}
```

Comment:
this does the same thing

Reply:
by that i mean it will still insert a new style into the document, not really saving much

Reply:
inline styles would though? style={{gap: space(wrapperGap)}} since those are not via emotion

Reply:
That makes sense, will change to inline styles.

I was going off [this section](https://develop.sentry.dev/frontend/using-styled-components/#use-style--css-attributes) in the docs. We should maybe update it to say that styles are better than css insertions if that is the case.